satellite/satellitedb/irreparable.go

```diff
@@ -108,13 +71,13 @@ func (db *Database) Get(ctx context.Context, segmentPath []byte) (resp *RemoteSe
 }
 
 // Delete a irreparable's segment info from the db
-func (db *Database) Delete(ctx context.Context, segmentPath []byte) (err error) {
+func (db *irreparable) Delete(ctx context.Context, segmentPath []byte) (err error) {
 	_, err = db.db.Delete_Irreparabledb_By_Segmentpath(ctx, dbx.Irreparabledb_Segmentpath(segmentPath))
 
 	return err
 }
 
-// Close close db connection
-func (db *Database) Close() (err error) {
+// Close  close db connection
+func (db *irreparable) Close() (err error) {
```

Comment:
Method `Close` will be triggered for master db not for separate elements. Should be removed.

Reply:
done